packages/ag-charts-community/src/chart/series/seriesEvents.ts

```diff
@@ -0,0 +1,21 @@
+import type { DataModel, ProcessedData, PropertyDefinition } from '../data/dataModel';
+
+export type SeriesEventType = 'data-update' | 'processData-prerequest' | 'visibility-changed';
+
+export interface BaseSeriesEvent<_T extends SeriesEventType> {}
+
+export interface SeriesDataUpdateEvent extends BaseSeriesEvent<'data-update'> {
+    dataModel: DataModel<any, any, any>;
+    processedData: ProcessedData<any>;
+}
+
+export interface SeriesPrerequestDataEvent extends BaseSeriesEvent<'processData-prerequest'> {
+    props: PropertyDefinition<string>[];
```

Comment:
Just wondering if `string` is the right generic type here - maybe `unknown` would be better if feasible?

Reply:
I'll try that.

Reply:
`unknown` works if I update errorBar.ts to reflect this. Done.